drill/test_drill.py

```diff
@@ -1,10 +1,12 @@
 import os
+import sys
 import unittest
 
 from parameterized import parameterized
-
 from integration_tests.dataproc_test_case import DataprocTestCase
 
+FLAGS = flags.FLAGS
```

Comment:
Delete?

Reply:
FLAGS is used within the test_drill function.